app/src/main/java/io/appium/uiautomator2/server/ServerConfig.java

```diff
@@ -4,6 +4,25 @@
 public class ServerConfig {
     private final static int PORT = 6790;
 
+    private static boolean allowInvisibleElements = false;
```

Comment:
I think adding them to [session](https://github.com/appium/appium-uiautomator2-server/blob/8774414fdd6b5b3b846039af12de78c9bd3b2bbd/app/src/main/java/io/appium/uiautomator2/model/Session.java#L15) would be a better approach, because down the line if there is any new setting to add, then we need to add that attribute and also need to corresponding setter and getter methods. Instead, if we can add them to session then it will minimize the effort.